packages/xo-server/src/xapi/index.js

```diff
@@ -779,7 +726,7 @@ export default class Xapi extends XapiBase {
     const exportedNameLabel = vm.name_label
     if (!vm.is_a_snapshot) {
       if (!bypassVdiChainsCheck) {
-        this._assertHealthyVdiChains(vm)
+        this.assertHealthyVdiChains(vm.$ref)
```

Comment:
There is no such method, `@xen-orchestra/xapi` has an async `VM_assertHealthyVdiChains` method though.